controllers/inventory-controller.js

```diff
@@ -3,7 +3,35 @@ const knex = require("knex")(require("../knexfile"));
 const index = async (_req, res) => {
   try {
     const data = await knex("inventories");
-    res.status(200).json(data);
+    const dataWarehouse = await knex("warehouses");
+
+    const inventoryData = data.map((item) => {
+      const {
+        id,
+        warehouse_id,
+        item_name,
+        description,
+        category,
+        status,
+        quantity,
+      } = item;
+
+      const { warehouse_name } = dataWarehouse.find(
+        (w) => w.id === warehouse_id
```

Comment:
w is hard to tell. Feel free to always be descriptive.
` (warehouse) => warehouse.id === warehouse_id`